src/cmd/tests/load_config_test.rs

```diff
@@ -137,6 +139,7 @@ fn test_load_frontend_example_config() {
                 remote_write: Some(Default::default()),
                 ..Default::default()
             },
+            grpc: GrpcOptions::default().with_hostname("127.0.0.1"),
```

Comment:
"127.0.0.1:3001"?

Reply:
Resolved in [3d31b31](https://github.com/GreptimeTeam/greptimedb/pull/5314/commits/3d31b3109a15cac2ab0c8a5f116df5f5300d5f52),
Should be "127.0.0.1:4001" for frontend